sdk/loadtesting/azure-developer-loadtesting/CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Release History
 
+## 1.1.0b1 (2025-03-24)
```

Comment:
Nit you should also update the date before releasing

Reply:
Done